examples/one_dimensional_diffusion.jl

```diff
@@ -120,13 +130,36 @@ run!(simulation)
 # iterations we ended up saving at, and plot the evolution of the
 # temperature profile in a loop over the iterations.
 
-T = FieldTimeSeries("one_dimensional_diffusion.jld2", "T")
+T_timeseries = FieldTimeSeries("one_dimensional_diffusion.jld2", "T")
+times = T_timeseries.times
+
+n = Observable(1)
+
+T = @lift interior(T_timeseries[$n], 1, 1, :)
+
+label = @lift "t = " * string(round(times[$n], digits=3))
+
+fig = Figure()
+
+ax = Axis(fig[2, 1];
+          xlabel = "Temperature (ᵒC)",
+          ylabel = "z",
+          xlabelsize = 20,
+          ylabelsize = 20,
+          limits = ((0, 1), nothing))
+
+lines!(T, z; linewidth)
+
+fig[1, 1] = Label(fig, label, textsize=24, tellwidth=false)
+
+# Finally, we record a movie.
 
-anim = @animate for (i, t) in enumerate(T.times)
-    Ti = interior(T[i], 1, 1, :)
+frames = 1:length(times)
 
-    plot(Ti, z; linewidth, title=@sprintf("t = %.3f", t),
-         label="", xlabel="Temperature", ylabel="z", xlims=(0, 1))
+record(fig, "one_dimensional_diffusion.mp4", frames, framerate=8) do i
+    @info "Plotting frame $i of $(frames[end])..."
+    n[] = i
 end
+nothing #hide
 
-mp4(anim, "one_dimensional_diffusion.mp4", fps = 15) # hide
+# ![](one_dimensional_diffusion.mp4)
```

Comment:
this will be removed eventually, I presume?

Reply:
Nope it’s needed for the animation to pop up in the docs. Or I haven’t figured out a different way to do that.

Reply:
Thanks for reminding me!